### ISSUE TITLE
Relationship issue, if model1 has many model2, then model2 should be able to belong to model1 right?

### ISSUE THREAD

**Issue by zacharygoh** [svtslv/nestjs-objection#7]
![image](https://user-images.githubusercontent.com/6231521/108654834-78af3780-7504-11eb-8bc4-6c6c76c65c39.png)

It will have dependencies issue, I'm following the approach provided with the example.

**Comment by H256** [comment]
@zacharygoh We ran into the same Problem in the mono-repo mode. We solved it by using a fat-arrow function that returns the class type (constructor).

**tl;dr** use `modelClass: () => MyModel,`

Something along these lines might work.

```typescript
@Relation({
    modelClass: () => NodeModel,
    relation: Model.BelongsToOneRelation,
    join: {
      from: 'storages.node_id',
      to: 'nodes.id',
    },
  })
  node: NodesModel;
```

The problem was that in monorepo mode ObjectionJS tries to lazy load the model file using require() wich won't work because the file is compiled (webpacked) into `main.js` and thus not available anymore for require(). So, all three solutions given here wouldn't work.
https://vincit.github.io/objection.js/guide/relations.html#require-loops
